core/Cargo.toml

```diff
@@ -64,6 +64,7 @@ image = { version = "0.24.8", default-features = false, features = ["tiff", "dxt
 enum-map = "2.7.3"
 ttf-parser = "0.20"
 num-bigint = "0.4"
+unic-segment = "0.9.0"
```

Comment:
oh nyo: https://rustsec.org/advisories/RUSTSEC-2025-0098

https://github.com/ruffle-rs/ruffle/actions/runs/18682103319/job/53265402809?pr=21985#step:5:7043

Reply:
Is there a reason other than this lib not supporting current unicode standard? Note that this behavior does not match Flash, it's a Ruffle feature atm (not to mention Flash wouldn't work with new unicode rules anyway).

Reply:
According to evilpie in the issue where it was reported as unmaintained:

> The last commit has been 5 years ago. Outdated Unicode information causes compatibility issues. (See for example https://github.com/denoland/rust-urlpattern/issues/54)

Reply:
I am waiving them in the wgpu repo because we use them via `deno` and `urlpattern`. It appears that `urlpattern` 0.4 has a replacement, but we can't switch to that until deno does and we update deno.